types/shallow-equals/index.d.ts

```diff
@@ -0,0 +1,16 @@
+// Type definitions for shallow-equals v1.0.0
+// Project: https://github.com/hughsk/shallow-equals
+// Definitions by: Ross Solomon <https://github.com/rsolomon>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export = shallow_equals;
+
+declare function shallow_equals<A, B>(
```

Comment:
I considered making both `a` and `b` the same type `T`, but figured that might be too strict for some use cases.